<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomMapGenerator</class>
 <widget class="QMainWindow" name="RoomMapGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Map Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Band.of.Brothers.E02.Day.of.Days.720p.jpg</normaloff>../../../../../Band.of.Brothers.E02.Day.of.Days.720p.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>38</y>
      <width>419</width>
      <height>368</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Room List :</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>258</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="lineEditRoomList"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtobBrowseRoomList">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Image Area Map :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>258</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEditImageMap"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonBrowseImageMap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Destination :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>228</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="lineEditDestination">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDestination">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="minimumSize">
            <size>
             <width>321</width>
             <height>25</height>
            </size>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonGenerate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClose">
             <property name="text">
              <string>Close</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>progressBar</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>pushButtonClose</zorder>
   <zorder>pushButtonGenerate</zorder>
   <zorder>horizontalSpacer_5</zorder>
   <zorder>lineEditDestination</zorder>
   <zorder>horizontalSpacer_6</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButtonDestination</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>verticalSpacer_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtobBrowseRoomList</sender>
   <signal>clicked()</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>selectRoomListFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBrowseImageMap</sender>
   <signal>clicked()</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>selectImageMapFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditImageMap</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>enablePushButtonGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRoomList</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>enablePushButtonGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGenerate</sender>
   <signal>clicked()</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>createImageMapFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDestination</sender>
   <signal>clicked()</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>selectDestinationFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDestination</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RoomMapGenerator</receiver>
   <slot>enablePushButtonGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectRoomListFile()</slot>
  <slot>selectImageMapFile()</slot>
  <slot>enablePushButtonGenerate()</slot>
  <slot>createImageMapFile()</slot>
  <slot>selectDestinationFile()</slot>
 </slots>
</ui>
